test/fixtures/suit-nesting.css

```diff
@@ -0,0 +1,33 @@
+/** @define Component **/
+
+.Component {
+  color: green;
+
+  .Component-descendant {}
+  &.is-stateName {}
+  &-descendant {}
+  &-descendant--modifier {}
+  &--modifier {}
+  &:pseudo-class {}
+  &:pseudo-element {}
+  &::pseudo-element {}
+  &[attr] {}
+}
+
+.Component-standAloneDescendant {}
+
+.Component {
```

Comment:
Cool yeah. Perhaps in strict mode we can disallow it. So:

``` css
// bad
/** @define Component */
.Component .Component-descendant {}

// good
/** @define Component; weak */
.Component .Component-descendant {}
```

We should raise a separate issue, but in the meantime I'll remove it from the test fixtures in strict mode.